eng/build.ps1

```diff
@@ -14,21 +14,24 @@ Param(
   [string]$arch,
   [string]$subsetCategory,
   [string]$subset,
-  [string]$runtimeConfiguration,
-  [string]$librariesConfiguration,
+  [ValidateSet("Debug","Release","Checked")][string]$runtimeConfiguration,
+  [ValidateSet("Debug","Release")][string]$librariesConfiguration,
   [Parameter(ValueFromRemainingArguments=$true)][String[]]$properties
 )
 
 function Get-Help() {
   Write-Host "Common settings:"
-  Write-Host "  -subset                 Build a subset, print available subsets with -subset help"
-  Write-Host "  -subsetCategory         Build a subsetCategory, print available subsetCategories with -subset help"
-  Write-Host "  -os                     Build operating system: Windows_NT or Unix"
-  Write-Host "  -arch                   Build platform: x86, x64, arm or arm64"
-  Write-Host "  -configuration <value>  Build configuration: Debug or Release (short: -c)"
-  Write-Host "  -verbosity <value>      MSBuild verbosity: q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic] (short: -v)"
-  Write-Host "  -binaryLog              Output binary log (short: -bl)"
-  Write-Host "  -help                   Print help and exit (short: -h)"
+  Write-Host "  -subset                   Build a subset, print available subsets with -subset help"
+  Write-Host "  -subsetCategory           Build a subsetCategory, print available subsetCategories with -subset help"
+  Write-Host "  -vs                       Open the solution with VS for Test Explorer support. Path or solution name (ie -vs Microsoft.CSharp)"
+  Write-Host "  -os                       Build operating system: Windows_NT or Unix"
+  Write-Host "  -arch                     Build platform: x86, x64, arm or arm64"
+  Write-Host "  -configuration            Build configuration: Debug, Release or [CoreCLR]Checked (short: -c)"
```

Comment:
The OCD in me wants me to point out that some of this change uses "Debug, Release, Checked" and others use "Debug, Checked, Release". This is an ignorable comment :smile: